package.json

```diff
@@ -30,7 +30,7 @@
     "presystem-test": "npm run compile",
     "system-test": "mocha build/system-test --timeout 600000",
     "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
-    "test": "c8 mocha build/test",
+    "test": "c8 mocha build/test --recursive",
```

Comment:
We've got a change pending actually that will obsolete this, but I will deal with it when that gets merged.

Reply:
I just added it here to get CI to run the publisher tests too.